.golangci.yaml

```diff
@@ -15,7 +15,6 @@ linters:
     - typecheck
     - unused
     - gci
-    - exportloopref
```

Comment:
What's this do?

Reply:
this used to check that you *didn't* export a pointer to the (single place identity) loop var, prior to the change in loop variable semantics. this was the problem where if you built a slice of pointers to your loop var, or launched goroutines with such, they would all have the same pointer unless you wrote like `i := i` to re-shadow it as a fresh local identity every iteration (which is now the default).

now there's a new lint you can turn on that yells at you to go back and delete those lines :roll_eyes: